packages/playwright/src/runner/testServer.ts

```diff
@@ -418,7 +418,17 @@ export class TestServerDispatcher implements TestServerInterface {
       }
       return { config };
     } catch (e) {
-      return { config: null, error: serializeError(e) };
+      const error: reporterTypes.TestError = serializeError(e);
+      if (e.name === 'SyntaxError' && e.code === 'BABEL_PARSE_ERROR' && this._configLocation.resolvedConfigFile) {
```

Comment:
We should find a better place for this. At least in utils. Don't we have such code somewhere already? Perhaps `serializeError()` should do that automatically and always set `location`?